README.md

```diff
@@ -1,17 +1,17 @@
 [![Build Status](https://dev.azure.com/azure/template-analyzer/_apis/build/status/BuildAndTest?branchName=main)](https://dev.azure.com/azure/template-analyzer/_build/latest?definitionId=91&branchName=main)
 [![Code Coverage](https://shields.io/azure-devops/coverage/azure/template-analyzer/91)](https://dev.azure.com/azure/template-analyzer/_build/latest?definitionId=91&branchName=main)
 
-# Template Best Practice Analyzer (BPA)
+# Template Analyzer (TA)
```

Comment:
``` suggestion
# Template Analyzer
```

Reply:
Do you think leaving TA could be helpful as the rule names are TA-XXXXX?

Reply:
I don't think having TA here gives any clarity to rule ids, since they aren't mentioned here.  Personally, I think TA in the rule id is self-explanatory (and we were going on that assumption anyway, since we used 'TA' and not 'BPA' 🙂).  Since the 'TA' acronym isn't used anywhere, it seems odd to introduce it.